walk-in-interview/src/main/webapp/new-job/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style.css">
+</head>
+
+<body>
+
+    <div class="header-container">
+        <input id="new-job-cancel">
+        <div id="new-job-page-title"></div>
+        <input id="new-job-submit">
+    </div>
+
+    <div class="content-container">
+        <div class="content-container">
+            <input class="job-field" id="new-job-title" />
+            <textarea class="job-field" id="new-job-description"></textarea>
+            <input class="job-field" id="new-job-address" />
+
+            <div class="job-field">
+                <div id="new-job-requirements-title"></div>
+                <ul id="new-job-requirements-list"></ul>
```

Comment:
maybe add a TODO here to fill in the requirement list in the future

Reply:
This is currently being rendered from the JS side so I've added the TODO there since there is no more work to be done in the HTML file.